lib/Error.js

```diff
@@ -1,142 +1,138 @@
-'use strict';
```

Comment:
Let's mark these files as generated code in .gitattributes : https://github.com/stripe-dev/api-codegen-next/blob/master/.gitattributes#L1


This will allow github to collapse changes in them by default.

Reply:
Done!